docs/core-features/table_lookups.md

```diff
@@ -1,5 +1,13 @@
 # Table lookup
 
+{% hint style="success" %}
+**Zama 5-Question Developer Survey**
+
```

Comment:
do we really want it here as well? it starts to be too much for me

Reply:
Ok I would still keep the banner on those most viewed pages but move it to the bottom of the page to make it less intrusive. Thanks!